src/cljs/centrebull/shooters/view.cljs

```diff
@@ -0,0 +1,54 @@
+(ns centrebull.shooters.view
+  (:require [centrebull.components.search :refer [search]]
+            [centrebull.components.input :refer [input]]
+            [re-frame.core :as rf]))
+
+(defn shooters-page [toggle-action]
+  [:section
+   [:card
+    [:h2 {:local "9/12"} "Shooters"]
+    [:button {:local "3/12" :on-click toggle-action} "New Shooter"]
+
+    [search "/shooters/search"
+     (fn [{:keys [shooter/sid shooter/preferred-name shooter/first-name shooter/last-name shooter/club]}]
+       [:div {:on-click #(rf/dispatch [:set-active-shooter sid])}
+        [:div {:local "1/3"} sid]
+        [:div {:local "1/3"} (if (empty? preferred-name) (str first-name " " last-name) preferred-name)]
```

Comment:
That's fine. Mabe move the calculated field of first-name + last-name into a let statement above.